cfpb/capital-framework
# Minor variable cleanup

`@fcm-bp-max` and `@fcm-bp-min` were removed in https://github.com/cfpb/capital-framework/commit/931149df0c40cf3e01e3b70a003b1acaf6f30206#diff-8db0ab7c481376d02c8ba276e4852714, but remained in the documentation.

## Removals

- Removes unused `@fcm-bp-max` and `@fcm-bp-min` variables.

## Changes

- Updates variables paragraph to apply to components that have a variety of variables, not only color and sizing.
- Alphabetizes variable sections.
- Align variable values per section.
- Remove unneeded top-level section in cf-icons.
- Added missing TOC entry in cf-typography.
- Re-generate yarn.lock and cache.

## Testing

1. Pull branch, run `cd docs && gulp && bundle exec jekyll serve`
2. Visit components on http://127.0.0.1:4000/components/ and see that variables table of contents links work.
